ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/highLevelHumanoidControl/factories/HighLevelHumanoidControllerFactory.java

```diff
@@ -576,7 +554,6 @@ public FrameVector3DReadOnly getCenterOfMassVelocity()
                                                                                   initialControllerState,
                                                                                   highLevelControllerParameters,
                                                                                   walkingControllerParameters,
-                                                                                  pushRecoveryControllerParameters,
```

Comment:
is it worth passing in the fast walking parameters this way (and with that adding fast walking parameters to WholeBodyControllerParameters and all that stuff), or does it make more sense to just keep it as a custom control state? Seems like the first option is a bit more invasive